cmd/okctl/delete.go

```diff
@@ -31,14 +31,16 @@ func buildDeleteCommand(o *okctl.Okctl) *cobra.Command {
 	deleteClusterCommand := buildDeleteClusterCommand(o)
 	cmd.AddCommand(deleteClusterCommand)
 	cmd.AddCommand(buildDeletePostgresCommand(o))
+	addAuthenticationFlags(cmd)
 
 	return cmd
 }
 
 // DeleteClusterOpts contains the required inputs
 type DeleteClusterOpts struct {
-	AWSCredentialsType    string
-	GithubCredentialsType string
+	AWSCredentialsType     string
+	GithubCredentialsType  string
```

Comment:
The credentials type variables here are not in use anymore?

Reply:
Removed these